### DISCUSSION TITLE
How to call a script after `babel watch` command

### DISCUSSION THREAD

**Discussion by federico-qadra** [babel/babel#12951] [Q&A] [answered]
Hello,

I need to execute a command at the end of each build during `babel --watch`.
I'm actually doing a similar thing in another project built with TSDX doing this: `tsdx watch --onSuccess "echo test"`, but I can't find a way to do it in Babel.

Is there any options to execute a command on a babel watch success?

Thank you

**Comment by nicolo-ribaudo** [comment, marked as answer]
It's not possible, the `babel` CLI is designed to be only used in simple cases.
I suggest using a task runner such as `gulp`, so that you can customize it as much as you want.

**Reply by federico-qadra** [reply]
Oh, I see. My command was pretty simple, something like `babel src -d . --watch --source-maps --copy-files`, adding `gulp` will add a bit of effort to solve a small issue.

Thanks anyway, cheers
